gnd/src/main/java/com/google/android/gnd/ui/offlinearea/OfflineAreasFragment.java

```diff
@@ -50,17 +52,26 @@
   public void onCreate(@Nullable Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
     viewModel = getViewModel(OfflineAreasViewModel.class);
-    // TODO: use the viewmodel
+    this.offlineAreaListAdapter = new OfflineAreaListAdapter();
+    viewModel.getOfflineAreas().observe(this, offlineAreaListAdapter::update);
```

Comment:
Moving these two lines to the end of `onCreateView` would allow `offlineAreaListAdapter` to be declared as local variable. The adapter isn't being used anywhere else. Wdyt?

Reply:
done! 34b3dfa